MeetingNotes.md

```diff
@@ -1,81 +1,124 @@
-  3. Sean suggests making a poll and curating our speakers to the interests of the members.
-  4. All organizers have access to the twitter account?
-    1.	Make sure that all of us can be responsible for what is said and what we follow/retweet.
-2. Tech Series (Speakers Jan, Feb, March)
-  1. Thinking of doing a series of talks instead of just standard talks
-  2. No single person speaking who is an “expert,” more of a panel discussion
-  3. More of a book club
-  4. Alex Hoffman wanted to do a Javascript talk in February.
-  5. Docker workshop/series
-3. Workshops before the summer
-  1. Workshops last year were in the summer, didn’t’ have a great turnout
-  2. Make sure that we get the workshops out before summer for greater engagement.
-  3. Consistency and focus of topics
-4. Member engagement
-  1. Putting together a draft for the poll of members and the topics that they would like to learn about.
-    1. Example: http://stackoverflow.com/research/developer-survey-2016
-  2. 13th and L meetup location? We can learn more about the preference of members for location change based on the poll.
-  3. Sponsors
-    1. Engage more companies, transparency of how its being organized?
-    2. Public slack channel where we can discuss how the meetup is being organized?
-5. Diversity
-  1. Pat is doing research and is going to be reaching out to different members of the community to try and get speakers outside of our existing pool.
-
-### Feb 4th 2017
-1.	We need to figure out what we’re going to do for April, May, June.
-  1.	Schedule the regular meetup first
-  2.	Workshops are a bonus
-  3.	Sahil needs to reach out to someone in April: Bjorn Westergaard
-  4.	Bringing in known speakers?
-    1.	Coming up with a list of “known” speakers for April, May, June
-      1.	Jeff Harcourt
-      2.	Kate Rabinowitz
-      3.	CTO 1776
-      4.	Capitol One – James Patterson
-      5.	USDS - Robbie Holmes
-      6.	Roy Osherove
-      7.	Reginald Braithwaite
-      8.	Nathan Hessler
-2.	Monitoring/polling the members to make sure we’re having the right amount of events
-3.	Half of those who attend are new members, how to make them repeat members?
-4.	Diversity
-  1.	Pat to reach out to Taylor and Lougenia
-5.	Tech Talk DC Happy Hour 
-  1.	After a meetup or as a regular event.
-
+- - - -
+### June 28th 2017
+1. Retrospective on Last month's event
+    1. Kevin Groat's talk on React/Redux
+    2. Well attended (40 people/80 people responded)
+    3. Topic was on Wyn's Tech Radar
+    4. Pros:
+        1. No attendance limit may have helped (continue going forward)
+        2. Low rate of attrition (we expect 33%, we got high 40%'s)
+    5. Cons:
+        1. Internet went down
+2. Upcoming events
+    1. Nara Kasbergen - Empathy as a Service
+    2. Identified from WeSpeakToo.org
+    3. NPR developer
```

Comment:
Oh I get it, you thought all of them were different bullets. No this is just detailing July's talk.  I didn't want to list topics we didn't approve/merge.